toolbox/linux/nowwhat.rst

```diff
@@ -0,0 +1,127 @@
+I Installed It!  Now What?
+==========================
+
+Congratulations! If everything worked properly, you should be looking at a
+freshly installed Linux login screen, either inside a virtual machine or "for
+real for real" on your main computer.
+
+What to do from here is up to you. The Toolbox pages after this one give some
+great things to learn to do on Linux. However, here are some other things you
+may want to do besides the technical stuff.
+
+*Note that the commands in code blocks below should be run in your Terminal
+program on Linux.  Also, these assume you installed Ubuntu.*
+
+How Do I Install Stuff?
+-----------------------
+
+Every Linux distribution has its own package manager, and so the instructions
+may be slightly different depending on your distro. If you did Ubuntu, all you
+need to do is run:
+
+.. code::
+
+   sudo apt install <package name>
+
+Typically the package name is exactly what you'd expect, but lower case. For
+example, ``git``, ``python``, ``firefox``, etc. However, sometimes it may have a
+slighly different name from expected. You can search for packages `online
+<http://packages.ubuntu.com>`_.
+
+Sometimes, software that you want may not be packaged, but their site may offer
+Linux downloads in ``.deb`` or ``.rpm`` format. In these cases, you can download
+the ``.deb`` (not the RPM!) and double click that to install it.
+
+Is There An Easier Way to Install Stuff?
+----------------------------------------
+
+Yeah, there's something called the Ubuntu Software Center (you should be able to
+find it in the menu). You can use that to search and install programs, similar
+to the Mac/Windows app stores. However we'd recommend that you get used to a
+command-line package manager.
+
+Where's Google Chrome?
+----------------------
+
+One common one that may trip you up is Chrome. In the Linux world, there are two
+versions of Chrome: *Google Chrome* and *Chromium*. *Chromium* is an open source
+web browser project that *Google Chrome* is based on. As such, it is almost
+always in your package manager as ``chromium``. Google Chrome is simply Chromium
+plus a built-in PDF reader, a Flash plugin, and a Google-colored icon. It's
+easiest to install Chromium by simply doing:
+
+.. code::
+
+   sudo apt install chromium
```

Comment:
`apt install` is [right](http://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get) :P